tests/io/test_data_catalog.py

```diff
@@ -652,3 +652,14 @@ def test_replacing_nonword_characters(self):
         assert "ds2_spark" in catalog.datasets.__dict__
         assert "ds3__csv" in catalog.datasets.__dict__
         assert "jalapeño" in catalog.datasets.__dict__
+
+    def test_no_version_cloud(self):
+        """Check the error if no versions are available for load from cloud storage"""
+        version = Version(load=None, save=None)
+        ds = CSVDataSet("s3://bucket/file.csv", version=version)
+        pattern = re.escape(
+            f"Did not find any versions for {ds} "
+            f"This could be due to insufficient permission."
+        )
+        with pytest.raises(DataSetError, match=pattern):
+            ds.load()
```

Comment:
```python
    def test_no_versions(self, versioned_plot_writer):
        """Check the error if no versions are available for load."""
        pattern = r"Did not find any versions for MatplotlibWriter\(.+\)"
        with pytest.raises(DataSetError, match=pattern):
            versioned_plot_writer.load()
```

I found this test in one of our datasets test, can we follow the same pattern here? I think "rf" should be the same as escape.

Reply:
I tried using "rf" but it doesn't seem to work.